librz/bin/dwarf/dwarf.c

```diff
@@ -991,40 +991,35 @@ RZ_IPI bool buf_read_block(RzBuffer *buffer, RzBinDwarfBlock *block) {
  * \brief Parses attribute value based on its definition
  *        and stores it into `value`
  */
-RZ_IPI bool attr_parse(RzBuffer *buffer, RzBinDwarfAttr *value, DwAttrOption *in) {
+RZ_IPI __attribute__((optimize("O0"))) bool attr_parse(RzBuffer *buffer, RzBinDwarfAttr *value, DwAttrOption *in) {
```

Comment:
Add a comment about why, preferably with a link to GCC bug. Could you open a new bug in GCC?